src/pal/src/file/file.cpp

```diff
@@ -2999,8 +2999,11 @@ OUT  PLARGE_INTEGER lpFileSize)
             &dwFileSizeHigh
             );
 
-        lpFileSize->u.LowPart = dwFileSizeLow;
-        lpFileSize->u.HighPart = dwFileSizeHigh;
+        if (NO_ERROR == palError)
```

Comment:
This one was caused by assigning uninitialized `dwFileSizeLow` to `lpFileSize->u.LowPart` in case the call to `InternalGetFileSize` failed. In that case, the `InternalGetFileSize` doesn't set its output parameter and so the `dwFileSizeLow` is a garbage.